flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -103,37 +119,68 @@ def run(argv=None, save_main_session=True):
     with beam.Pipeline(options=options) as p:
 
         # Read from BigQuery into a PCollection.
-        messages = (
-            p
-            | 'Read from BigQuery Table' >>
-            beam.io.ReadFromBigQuery(
-                table=known_args.input_table
+        if known_args.query is not None:
```

Comment:
I'd suspect using a public dataset is not the normal case.  we're using it to help with providing a large dataset example, but I'd expect this branch to always be taken.  

suggestion is to consider "input_table" as the primary way to read BQ;